tests/acceptance/blocking_adapter_test.py

```diff
@@ -476,13 +469,13 @@ def test(self):
         self.assertIsInstance(method, pika.spec.Basic.GetOk)
         self.assertEqual(method.delivery_tag, 1)
         self.assertFalse(method.redelivered)
-        self.assertEqual(method.exchange, _p3_as_bytes(''))
-        self.assertEqual(method.routing_key, q_name)
+        self.assertEqual(method.exchange, as_bytes(''))
+        self.assertEqual(method.routing_key, as_bytes(q_name))
```

Comment:
@everilae: according to the RabbitMQ folks on this page https://www.rabbitmq.com/blog/2014/04/02/breaking-things-with-rabbitmq-3-3/:
 > The AMQP 'shortstr' data type (which is used for things like exchange and queue names, routing keys and so on) is defined by the spec as being in UTF-8 format.

If they are correct, it means that all of these values should be going in as UTF-8 strings (str would be okay in python2.x, too) and combing back as UTF-8 strings, but not as bytes.

CC @gmr